frontend/.eslintignore

```diff
@@ -1,2 +0,0 @@
-node_modules/
-build/
```

Comment:
Why is this gone? Are those excluded by default?

Reply:
`node_modules` is; `build` is excluded in `excludePatterns` in the config. I thought having fewer dot-files in the root would be nice.

Reply:
Absolutely!